### ISSUE TITLE
⌨ Accessibility - My Profile - screen reader reads out some text twice

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#5007]
## ❌ Current behaviour (issue)
Some of the text in the My Profile page is being read twice when using Voiceover on Chrome.  See video below:

https://user-images.githubusercontent.com/1794660/205418330-34bcd6aa-0553-4013-9d22-7c82896ae077.mp4

### How to reproduce
1. Environment (prod or UAT): prod, talent.canada.ca/en/users/.../profile
2. Which browser and/or Assistive Technology? Voiceover on Mac - Chrome browser

**Step-by-step instructions explaining how to reproduce the problem.**

1. Enable voiceover
2. Make sure that you've filled out the "about me" and "language information" sections of the page
3. navigate through the page until you get to the text in the "about me" section.
4. Use voiceover key + right arrow key to read out the text.

Notice that some of the same text is read twice.

### Acceptance Criteria

- [ ] Ensure text is read once and correctly.
- [ ] Remove `<br>` tags in between the text items and just encompass them in `<p>` tags.  This seems to make the text read correctly. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns
- [ ] Check across components in My Profile for any similar `<br>` tags and replace these as well.
- [ ] Ensure there is no visual difference after changing any html/css structure.

**Comment by patcon** [comment]
@esizer mentioned that a quick check left him unable to reproduce

@patcon suggested it might have to do with breakpoints and screen width
